backend/api/Controllers/ParcelController.cs

```diff
@@ -67,6 +67,7 @@ public IActionResult GetParcels()
         /// <returns></returns>
         [HttpPost]
         [HasPermission(Permissions.PropertyView)]
+        [Route("/api/parcels/filter")]
```

Comment:
Can't have two posts with the same route. @Fosol let me know if this is a breaking change.

Reply:
Should be fine, these aren't used yet.

Reply:
I understand that there can be no two POST on the same route. But with this change it is now confusing to the API consumer how to filter parcels. There's now two (different) routes to do it:

- GET `/api/parcels?field1=value1&field2=value2`
- POST `/api/parcels/filter/` with a body of  `{ field1: value1, ...}`

I know the POST is there for convenience but I rather remove the [HttpPost] attribute and let it be a GET only operation. Note that I'm not saying we discard the whole `GetParcels(filter)` function, just not exposing it via HTTP

Reply:
There's a lot of debate out there on what is "right", I only ask we remain consistent, ideas?

- https://stackoverflow.com/questions/19637459/rest-api-using-post-instead-of-get
- https://stackoverflow.com/questions/46585/when-do-you-use-post-and-when-do-you-use-get
- https://www.moesif.com/blog/technical/api-design/REST-API-Design-Filtering-Sorting-and-Pagination/

Reply:
I think this is outside the scope of this review since I think you had the same issue with the GET/POST before I was forced to rename it. Feel free to discuss that here but I'm not planning on changing anything at this time.